qiskit/transpiler/preset_passmanagers/common.py

```diff
@@ -422,7 +426,12 @@ def generate_translation_passmanager(
                 method=unitary_synthesis_method,
                 target=target,
             ),
-            HighLevelSynthesis(hls_config=hls_config),
+            HighLevelSynthesis(
+                hls_config=hls_config,
+                coupling_map=coupling_map,
+                target=target,
+                use_qubit_indices=False,
```

Comment:
We can set this `True` here I think because we've applied a layout and routed the circuit, so if there are any higher level objects to synthesize here, their qargs are physical qubits in the target/coupling map.

```suggestion
                use_qubit_indices=True,
```